dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -303,6 +303,10 @@ private long countAllPublished() {
         return Arrays.asList(this.currentSession().getNamedQuery("Entry.findWorkflowsDescriptorTypes").setParameter("entryId", entryId).getSingleResult().toString());
     }
 
+    public List<Entry> findAllGitHubGenericEntriesWithNoTopic() {
```

Comment:
Entry is the parent entity of workflows/tools/etc, so the descriptor 'Generic' is redundant and could confuse.  Remove from names of method and query?